src/graphics/Drawables.cpp

```diff
@@ -231,7 +231,7 @@ void Lines::SetData(const Uint32 vertCount, const vector3f *vertices, const Colo
 	m_refreshVertexBuffer = true;
 
 	// if the number of vert mismatches then clear the current vertex buffer
-	if( m_vertexBuffer.Valid() && m_vertexBuffer->GetVertexCount() != vertCount ) {
+	if( m_vertexBuffer.Valid() && m_vertexBuffer->GetSize() != vertCount ) {
```

Comment:
Shouldn't this match the multiple colors version below with a capacity check? I'm confused.

Reply:
Yes it absolutely should. 
Not all these got changed because I did a search/replace for GetVertexCount() -> GetSize() and then did a highly scientific way of finding where buffer allocations occurred by sticking a breakpoint in the constructor and waiting until it got hit :)